src/SFA.DAS.Reservations.Infrastructure/TagHelpers/ExternalUrlHelper.cs

```diff
@@ -53,6 +53,11 @@ public string GenerateAddApprenticeUrl(Guid reservationId, string accountLegalEn
                 queryString += $"&courseCode={courseId}";
             }
 
+            if (string.IsNullOrWhiteSpace(courseId) && !startDate.HasValue)
```

Comment:
Bit picky, but can this be a method instead or property that describes what it is
```suggestion
            private IsLevyAccount => string.IsNullOrWhiteSpace(courseId) && !startDate.HasValue;
            
            if (IsLevyAccount)
            
            
```

Reply:
sure